underfs/s3a/src/test/java/alluxio/underfs/s3a/S3AUnderFileSystemTest.java

```diff
@@ -97,4 +88,18 @@ public void renameOnAmazonClientException() throws IOException {
     boolean result = mS3UnderFileSystem.renameFile(SRC, DST);
     Assert.assertFalse(result);
   }
+
+  @Test
+  public void createCredentials() throws Exception {
+    Map<PropertyKey, String> conf = new HashMap<>();
+    conf.put(PropertyKey.S3A_ACCESS_KEY, "key1");
+    conf.put(PropertyKey.S3A_SECRET_KEY, "key2");
+    try (Closeable c = new ConfigurationRule(conf).toResource()) {
+      UnderFileSystemConfiguration ufsConf = UnderFileSystemConfiguration.defaults();
+      AWSCredentialsProvider credentialsProvider =
+          S3AUnderFileSystem.createAwdCredentialsProvider(ufsConf);
+      Assert.assertEquals("key1", credentialsProvider.getCredentials().getAWSAccessKeyId());
+      Assert.assertEquals("key2", credentialsProvider.getCredentials().getAWSSecretKey());
```

Comment:
Added. Meanwhile, I think the type of  `credentialsProvider` to be `StaticCredentialsProvider` or `DefaultAWSCredentialsProviderChain` is implementation detail that is subject to change and should be opaque to the tests. I propose to create `SystemEnvironmentVariableRule` using techniques similar to https://stackoverflow.com/questions/318239/how-do-i-set-environment-variables-from-java, for test purposes.